<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H17T_Settings</class>
 <widget class="QDialog" name="H17T_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QPushButton" name="Btn_Save_Settings">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Cancel_Settings">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>500</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>851</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_general">
    <attribute name="title">
     <string>General</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_database">
    <attribute name="title">
     <string>Database</string>
    </attribute>
    <widget class="QFrame" name="db_frame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>221</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="db_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="db_status_label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Status :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="debugLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Cancel_Settings</sender>
   <signal>clicked()</signal>
   <receiver>H17T_Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
